pages/about.tsx

```diff
@@ -12,9 +12,10 @@ export default function About(): JSX.Element {
 		<Layout>
 			<Container maxWidth="sm">
 				<Box my={4}>
-					<MuiLink href="/">Go to the main page</MuiLink>
+					<Link href="/">
+						<MuiLink style={{ cursor: 'pointer' }}>Go to the main page</MuiLink>
```

Comment:
Are the cursor styles needed? Either way not a big fan of the inline styles. JSS is the way to go normally: https://material-ui.com/styles/basics/